apps/mongodb/workload/mongo_loadgen.yml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mongo-loadgen
+spec:
+  template:
+    metadata:
+      name: mongo-loadgen
+      labels:
+        app: mongo-loadgen
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: mongo-loadgen
+        image: openebs/ansible-runner
```

Comment:
did you mean openebs/mongo-client here and openebs/ansible-runner in the subsequent file (run_litmus_test) ?

Reply:
@ksatchit  openebs/ansible-runner will be the part of litmus job image. Where as openebs/mongo-client image will be the part of loadgen job. And this image we will crate using docker file in openebs/tet-tools